tree_node.go

```diff
@@ -137,9 +138,10 @@ func (node *TreeNode) Copy() *TreeNode {
 		nilHash:      node.nilHash,
 		nilChildHash: node.nilChildHash,
 		path:         node.path,
+		nodeSize:     node.nodeSize,
```

Comment:
`nodeSize` means the size of current node, does not include its children, can we make it include children, so that we can save some calculation when GC?